WTraining/app/middlewares/apiInfo.js

```diff
@@ -0,0 +1,16 @@
+const pjson = require('../../package.json');
```

Comment:
Change the name of this file to api_info.js in accordance with the document of good practices :)